src/pages/Main/Main.jsx

```diff
@@ -1,7 +1,85 @@
 import React from 'react';
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import CardList from './CardList/CardList';
+import CardListNew from './CardList/CardListNew';
+import MainCarousel from './MainCarousel/MainCarousel';
+import MAIN_DATA from './CardList/MainData.jsx';
 
 const Main = () => {
-  return <article>Main</article>;
+  const [productList, setProductList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data);
+      });
+  }, []);
+
+  return (
+    <MainWrapper>
+      <MainCarousel />
+      <MainContent>
+        <Suggestion>
+          <Title>오늘의 상품 추천</Title>
+          <SuggestionImg>
+            {MAIN_DATA.map(item => {
+              return <CardList key={item.id} item={item} />;
+            })}
+          </SuggestionImg>
+        </Suggestion>
+        <TodayNew>
+          <Title>오늘 입고 됐어요 !</Title>
+          <TodayNewImg>
+            {productList.map(product => {
+              return <CardListNew key={product.id} product={product} />;
+            })}
+          </TodayNewImg>
+        </TodayNew>
+      </MainContent>
+    </MainWrapper>
+  );
 };
 
 export default Main;
+
+const MainWrapper = styled.div`
+  width: 100%;
+`;
+
+const MainContent = styled.div`
+  width: 1024px;
+  margin: auto;
+`;
+
+const Suggestion = styled.div``;
+
+const Title = styled.div`
+  margin: 70px 0 30px;
+  color: ${props => props.theme.darkGrey};
+  font-size: 25px;
+  font-weight: 700;
+`;
+
+const SuggestionImg = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 20px;
+  justify-content: center;
+  align-content: center;
+  margin: 0 auto;
+`;
+
+const TodayNewImg = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 40px;
+  justify-content: center;
+  align-content: center;
```

Comment:
```suggestion
  place-items : center;
```

grid를 쓸 때, 가운데 정렬을 하고 싶다면 이 스타일 속성을 사용해도 좋습니다!

Reply:
감사합니다!